### ISSUE TITLE
greedy_decorder no output

### ISSUE THREAD

**Issue by yannhan** [HawkAaron/RNN-Transducer#10]
Hi, for testing part, the greedy_decorder or beam search gives a empty output. Below is the corresponding screens.
![Screenshot from 2019-04-15 17-11-02](https://user-images.githubusercontent.com/43621476/56168780-ce7a3680-5fa1-11e9-9088-6cad8b93ff4a.png)
![Screenshot from 2019-04-15 17-09-49](https://user-images.githubusercontent.com/43621476/56168782-d1752700-5fa1-11e9-85f5-0172096cbda2.png)

**Comment by HawkAaron** [comment]
Have you changed the blank label index ? 
You'd better check [the prediction](https://github.com/HawkAaron/RNN-Transducer/blob/graves2013/model.py#L92).

**Comment by yannhan** [comment]
I didn't change, the blank label index is 0. Do I need to change? I checked the prediction, is all 0.

**Comment by yannhan** [comment]
This is my phone map, I used character based map. The first should be the blank, index is 0.
{'': 0, 'T': 1, 'U': 2, 'S': 3, 'N': 4, 'F': 5, 'A': 6, 'C': 7, 'H': 8, 'D': 9, 'R': 10, 'W': 11, 'V': 12, 'J': 13, 'L': 14, 'Y': 15, ' ': 16, 'M': 17, 'G': 18, 'I': 19, 'O': 20, 'K': 21, 'E': 22, 'B': 23, '<sos>': 24}

**Comment by HawkAaron** [comment]
It seems that your model didn't converge, the training loss is too large, you can try other optimizers like adam.
